faster_sam/dependencies/schemas.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+from pydantic import BaseModel, Base64UrlStr, Field
+
+
+class SQSInfo(BaseModel):
+    id: str
+    body: str
+    receive_count: int
+    sent_timestamp: int
+    message_attributes: dict[str, str] | None
```

Comment:
Anotações de tipo para versões de Python abaixo de 3.10 não aceitam utilizar coleções built-in, precisa importar do módulo `typing` as classes para anotação como `Dict`. `List`, `Tuple`, etc.

Outro ponto é que não é aceito o operador de união para anotação de tipos, precisa utilizar `Union` ou  neste caso `Option`.